README.md

```diff
@@ -27,7 +27,7 @@ func TestFoo(t *testing.T) {
     })
 }
 ```
-The library provides the Equals, CmpEquals, DeepEquals, ErrorMatches,
+The library provides the Equals, CmpEquals, DeepEquals, Matches, ErrorMatches,
```

Comment:
It seems a bit of a shame that we have to keep this up to date with all the implemented checkers. Couldn't we just say it provides some checkers, but more may be added?